mica/stats/mag_stats/catalogs.py

```diff
@@ -0,0 +1,228 @@
+import os
```

Comment:
We've got a mica.catalog area.  Seems like this code might belong there anyway.

Reply:
Though if we go that way and move to a more general-purpose method, it seems like just having the methods pass content around just make one big answer of everything might be the way to go instead of the globals.